### PR TITLE
Dev/galera helm

### PR DESCRIPTION
**Pull request by PetkoVasilev**
This is the galera helm integration

### CODE REVIEW

File: helm-chart/config/galera.cnf [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+innodb_buffer_pool_size=122M
+innodb_doublewrite=1
+wsrep_debug=ON
+wsrep_on=ON
+wsrep_provider=/usr/lib/libgalera_smm.so
+wsrep_provider_options="gcache.size=300M; gcache.page_size=300M"
+wsrep_cluster_name="{{ .Values.LABEL }}-cluster"
+wsrep_cluster_address="gcomm://<<CLUSTER_ADDRESS>>"
+wsrep_sst_method=mariabackup
+wsrep_sst_auth = {{ .Values.ADMIN_USERNAME }}:{{ .Values.ADMIN_PASSWORD }}
```
Comment (alexander-aldev) [inline helm-chart/config/galera.cnf]: these should be expanded at runtime only, so probably <<ADMIN_USERNAME>> and <<ADMIN_PASSWORD>> need to be used?

Comment (PetkoVasilev) [reply helm-chart/config/galera.cnf]: hmm, yep, I missed those

**Comment by PetkoVasilev** [comment]
Fixed the helm error - minor formatting problems with the yaml files.
And prepared for merge with master.

**Comment by PetkoVasilev** [comment]
closing, since there is a newer version with backup

**Pull request closed without merging**